views/js/apipCreator/editor/inclusionOrderListing.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ *
+ */
+define([
+    'lodash',
+    'jquery',
+    'tpl!taoQtiItem/apipCreator/tpl/inclusionOrderListing/list'
+], function (_, $, listTpl) {
+    'use strict';
+    
+    var _ns = '.inclusion-order-listing';
+    
+    function render($container, elements){
+        
+        var i = 0;
+        var _elements = _.map(elements, function(element){
+            i++;//starting from 0
+            return {
+                id : element.id,
+                qti : element.qti,
+                content : element.content,
+                order : i
+            };
+        });
+        
+        $container.empty().append(listTpl({
+            elements:_elements
+        }));
+        var $sortable = $container.children('.order-list');
+        $sortable.sortable({
+            axis : 'y',
+            handle : '.content',
+            change : function(e, ui){
+                
+                var $helper = $(ui.helper);
+                var i = 1;
+                var qtiOrder = [];
+                var aeOrder = [];
+                
+                $sortable.children('.order-element').not('.ui-sortable-helper').each(function(e){
+                    
+                    var $li = $(this);
```

Comment:
The same thing here. I suggest use first callback function parameter as index instead of incrementing `i` variable:
```
.each(function(i){
    ...
    $li.data('order', i);
    $li.find('.order').html(i);
    ...
});
```